lesson_3/phone_book.py

```diff
@@ -0,0 +1,161 @@
+# Реализовать интерактивное меню с двумя опциями выбора:
+# а) добавление в справочник
+# б) вывод номера телефона по имени
+# в) вывод имени по номеру телефона
+#
+# Пример
+# Add to dict
+# Get telephone
+# Get name
+# 	1
+# 	John, +380953336667
+# 	1
+# 	Alan, +36674490
+# 	2
+# 	Alan
+# 	+36674490
+# 3
+# +380953336667
+# John
+# Посчитать количество номеров телефонов в предыдущем примере, в которых есть 3 или более одинаковых цифр подряд
+#
+#
+# Добавить возможность удаления человека из справочника
+# Добавить возможность редактирования номера\имени
+import re
+
+
+def get_number(question_string='? :'):
+    h = ''
+    while not h.isnumeric():
+        h = input(question_string)
+        if h.isnumeric():
+            return int(h)
+        elif h == '':
+            return 0
+        print('is not numeric \nplease put a number')
+
+
+def get_phone(question_string='Phone :'):
+    h = ''
+    while not h.isnumeric() or h[:1] == '+' and h[1:].isnumeric():
```

Comment:
create separate function: is_phone_number
```suggestion
    while not is_phone_number(h):
```